documentation/BookAnthology.json

```diff
@@ -131,8 +131,5 @@
       "approvalStatus" : "APPLIED",
       "applicationCode" : "123123"
     } ]
-  } ],
-  "publisherId" : "http://example.org/org/123",
```

Comment:
These fields were used for indexing in the database and we do not need them anymore